debian/preinst

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright 2015, EMC, Inc.
+
+set -e
+
+#############################################
+#  Check NodeJS is greater than 4.x
+#############################################
+check_nodejs(){
+    installed=0;
+    if [ "`type -p nodejs`" != "" ]; then
+        # Node JS being installed
+        NODEJS_VER=$(nodejs --version | sed s/v//)
+        if [ "$NODEJS_VER" \< "4.0" ] ; then
```

Comment:
Currently, we only support nodejs 4.x, we don't promise RackHD works for the version 5.x, 6.x, ...